web/cypress/integration/crd.spec.js

```diff
@@ -0,0 +1,37 @@
+describe('CRD', () => {
+    before(() => {
+        cy.visit('/');
+    })
+})
+
+it('loads CRD', () => {
```

Comment:
Qthis test would not have caught the CRD path issue.  CRDs added after Octant booted worked fine.